### ISSUE TITLE
Submitting form with "Enter" creates two records

### ISSUE THREAD

**Issue by romaninsh** [atk4/ui#173]
After much testing, I noticed that the problem ONLY OCCURS if there is a single field in a form.

``` php
require 'vendor/autoload.php';

class Brand extends \atk4\data\Model {
    public $table = 'brand';

    function init()
    {
        parent::init();

        $this->addField('name');
        $this->addField('producer');   // comment this out and it stops working!
    }
}

$db = new \atk4\data\Persistence_SQL('mysql:host=127.0.0.1;dbname=warehouse;charset=utf8', 'root', 'root');

$g = $layout->add(['CRUD']);
$g->setModel(new Brand($db));
```

Two requests are sent to the backend:

![screen shot 2017-06-24 at 16 32 56](https://user-images.githubusercontent.com/453929/27508997-e8010cd2-58fa-11e7-926c-10b48be07a2e.png)

and

![screen shot 2017-06-24 at 16 33 21](https://user-images.githubusercontent.com/453929/27508996-e2d6a6d6-58fa-11e7-8d89-a5619309206d.png)

With two fields only one request is sent:

![screen shot 2017-06-24 at 16 34 30](https://user-images.githubusercontent.com/453929/27509004-13c0b98a-58fb-11e7-9c95-62656218bf8a.png)

MAYBE it's because of error 500 ?

**Comment by romaninsh** [comment]
Unable to reproduce on develop...

**Comment by DarkSide666** [comment]
It's because of how browser treats 1 field forms. Some browsers send submit when you press Enter in 1-field form, but some doesn't.

Some reading:
https://stackoverflow.com/a/1370314/1466341
https://stackoverflow.com/a/39603896/1466341
https://stackoverflow.com/a/17797230/1466341

**Comment by romaninsh** [comment]
this native submit should be disabled.

**Comment by acicovic** [comment]
Hasn't been this solved?
If yes let's close the issue.

**Comment by DarkSide666** [comment]
This should be solved now by #379.
If problem will reappear then we can always create new github issue.
